app/main.py

```diff
@@ -1,8 +1,34 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += car.comfort_class * (self.clean_power - car.clean_mark)
+                car.clean_mark = self.clean_power
+        return round(income, 1)
+
+    def calculate_washing_price(self, car):
+        return round((car.comfort_class * (
+                    self.clean_power - car.clean_mark) * self.average_rating) / self.distance_from_city_center, 1)
```

Comment:
The `calculate_washing_price` method should include a check to ensure that the car's `clean_mark` is less than the station's `clean_power` before calculating the price. This prevents calculating a negative or incorrect price when the car is already cleaner than the station's capability.